website/content/docs/discovery/services.mdx

```diff
@@ -275,6 +275,255 @@ For Consul 0.9.3 and earlier you need to use `enableTagOverride`. Consul 1.0
+service. Tagged addresses can be used by remote agents and services as alternative
+addresses for communicating with the given node or service. Multiple tagged
+addresses address may be configured on a node or service.
+
+The syntax for defining a tagged address is as follows.
+
+<CodeTabs heading="Tagged address format">
+
+```hcl
+service {
+  name = "redis"
+  port = 80
+  tagged_addresses {
+    <tag> = {
+      address = "<address>"
+      port = port
+    }
+  }
+}
+```
+
+```json
+{
+  "service": {
+    "name": "redis",
+    "port": 80,
+    "tagged_addresses": {
+      "<tag>": {
+        "address": "<address>",
+        "port": port
+      }
+    }
+  }
+}
+```
+
+</CodeTabs>
+
+Consul supports several tagged address types. The various types and their use
+cases are detailed below.
+
+#### LAN
+
+LAN addresses are intended to be directly accessible only from services within
+the same Consul data center.
+
+##### Supported tags
+
+* `lan` - The IPv4 LAN address at which the node or service is accessible.
+* `lan_ipv4` - The IPv4 LAN address at which the node or service is accessible.
+* `lan_ipv6` - The IPv6 LAN address at which the node or service is accessible.
+
+<CodeTabs heading="Example LAN tagged address configuration">
+
+<CodeBlockConfig filename="redis-service.hcl">
+
+```hcl
+service {
+  name = "redis"
+  address = "192.0.2.10"
+  port = 80
+  tagged_addresses {
+    lan = {
+      address = "192.0.2.10"
+      port = 80
+    }
+    lan_ipv4 = {
+      address = "192.0.2.10"
+      port = 80
+    }
+    lan_ipv6 = {
+      address = "2001:db8:1:2:cafe::1337"
+      port = 80
+    }
+  }
+}
+```
+
+</CodeBlockConfig>
+
+<CodeBlockConfig filename="redis-service.json">
+
+```json
+{
+  "service": {
+    "name": "redis",
+    "address": "192.0.2.10",
+    "port": 80,
+    "tagged_addresses": {
+      "lan": {
+        "address": "192.0.2.10",
+        "port": 80
+      },
+      "lan_ipv4": {
+        "address": "192.0.2.10",
+        "port": 80
+      },
+      "lan_ipv6": {
+        "address": "2001:db8:1:2:cafe::1337",
+        "port": 80
+      }
+    }
+  }
+}
+```
+
+</CodeBlockConfig>
+</CodeTabs>
+
+#### Virtual
+
+Virtual tagged addresses are a logical address type which is used by
+[Connect](/docs/connect) to route traffic to the associated destination service
+in the service mesh when:
+
+1. [Transparent proxy](/docs/connect/transparent-proxy) is enabled.
+1. The service is not configured for individual instances to be
+   [dialed directly](/docs/connect/config-entries/service-defaults#dialeddirectly).
+
+-> **Note**: Virtual addresses are not required to be routable IPs within the
+network. They are strictly a control plane construct used to provide a single
+address for the instances of a given logical service.
+
+##### Supported tags
+
+The following address tag can be configured for a service to advertise the
+logical address to be used to dial it over the service mesh.
```

Comment:
```suggestion
Use the following address tag to specify the logical address the service should advertise when dialing over the service mesh.
```
I am not certain that I correctly restated the part about "dialing over the service mesh".

Reply:
```suggestion
Use the following address tag to specify the logical address at which the service can be reached by other services in the mesh.
```
Is this perhaps more clear?

Reply:
Yes, I think I was thrown off by the term 'dial' -- thinking it meant something more technical than literally just a service communicating with another another service